scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -316,6 +319,8 @@ Param (
 
     [string] $TwinUpdateFailureThreshold = $null,
 
+    [switch] $HostPlatform = $null,
```

Comment:
Please validate host platform is provided.

Reply:
I also did this in the runE2E.sh